cdn-server/cdn/src/index.ts

```diff
@@ -2,22 +2,32 @@ import { JWTVerifyResult, jwtVerify } from 'jose';
 import { Context, Env, Hono, Next, Schema } from 'hono';
 import { stream } from 'hono/streaming';
 
+export const signatureSha256Header = 'X-Signature-SHA256';
```

Comment:
Should the header be independent from the alg so that the user can use a different alg if they want?

Reply:
I see no reason why we should make it flexible right now. See my comment above.